src/pages/publication/my_publications/PublicationList.tsx

```diff
@@ -96,9 +97,23 @@ const PublicationList: FC<PublicationListProps> = ({ publications }) => {
                 <TableCell>
                   <NormalText>{new Date(publication.createdDate).toLocaleString()}</NormalText>
                 </TableCell>
+                <TableCell>
+                  {publication.status === PublicationStatus.PUBLISHED && (
+                    <Button
+                      color="primary"
+                      variant="outlined"
+                      component={RouterLink}
+                      to={`/publication/${publication.identifier}/public`}
+                      data-testid={`open-publication-${publication.identifier}`}>
+                      <MenuBookIcon />
+                      <StyledNormalTextWithIcon>{t('common:show')}</StyledNormalTextWithIcon>
+                    </Button>
+                  )}
+                </TableCell>
```

Comment:
Vet du at dette kun gjelder for publikasjoner som er published? Slik som jeg forstod det, så skulle det gå an å forhåndsvise publikasjoner i public mode også?

Reply:
Stemmer det, men tenkte å droppe det nå siden vi har en sjekk på at alle publikasjonene i `PublicPublications` må være published som må endres og dette bare var en "i farta"-PR. Men egentlig passer det ganske fint med en ren frontendoppgave som det akkurat nå